{
    "id": "patch-task-exception",
    "name": "Available for successful patch-read"
}